server/shared/storage/helpers.js

```diff
@@ -47,40 +39,14 @@ function processQuestion(element) {
 
 function processPrimitive(primitive) {
   if (!isPrimitive(primitive)) throw new Error('Invalid primitive');
-  if (!processor[primitive.type]) return Promise.resolve(primitive);
-  return processor[primitive.type](primitive);
+  return Promise.resolve(primitive);
```

Comment:
You can now entirely eliminate the 'default' processor.

Reply:
✅